static/src/javascripts-legacy/bootstraps/enhanced/main.js

```diff
@@ -163,10 +163,17 @@ define([
         }
 
         // use a #force-sw hash fragment to force service worker registration for local dev
-        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash === '#force-sw') {
+        if ((window.location.protocol === 'https:' && config.page.section !== 'identity') || window.location.hash.indexOf('force-sw') > -1) {
```

Comment:
I don't think `blockIAS` will be true until we're in enhanced though? So yes, the fetch will catch it, but `blockIAS` isn't persisted for the session? Thoughts @sndrs ?